### PR TITLE
post 5.3.100 pull

### PR DESCRIPTION
**Pull request by zuzunker**

### CODE REVIEW

**Review by SorinC6 — DISMISSED** [review]

**Review by zuzunker — COMMENTED** [review]
/check

File: packages/yoroi-extension/chrome/extension/background/handlers/utils.js [hunk 1]
```diff
@@ -186,16 +182,18 @@ async function getWalletState(publicDeriver: PublicDeriver<>): Promise<WalletSta
     throw new Error('unexpected missing asGetBalance result');
   }
   const balance = await canGetBalance.getBalance();
+  const rewardBalance = await canGetBalance.getBalance();
```
Comment (yushih) [inline packages/yoroi-extension/chrome/extension/background/handlers/utils.js]: @vsubhuman what's point of adding `rewardBalance`?

Comment (zuzunker) [reply packages/yoroi-extension/chrome/extension/background/handlers/utils.js]: @yushih , good catch! I was initially planning to have that change added and used, but then changed my mind, removed the usage of it, and forgot the declaration. The plan is to make this change later as a separate PR to review and test separately. Ultimate goal is to simplify and encapsulate as much wallet-depending APIs to be a part of the wallet itself as possible. In the particular case, it's a very common pattern when both the utxo balance and the reward balance need to be summed together to have the "total" wallet balance. Remember that `canGetBalance.getBalance()` returns only the utxo balance.

Comment (zuzunker) [reply packages/yoroi-extension/chrome/extension/background/handlers/utils.js]: I have removed this now

**Pull request merged**